src/features/import/components/ImportDialog/elements/ImpactSummary/AddedTags.tsx

```diff
@@ -69,7 +71,11 @@ const AddedTags: FC<AddedTagsProps> = ({
       </Typography>
       <Box display="flex" flexWrap="wrap" gap={1} paddingTop={1}>
         {addedTags.map((tag) => (
-          <TagChip key={tag.id} tag={tag} />
+          <TagChip
+            key={tag.id}
+            optionalString={` (${peoplePerTag[tag.id]})`}
+            tag={tag}
```

Comment:
How about this, to change the title directly?

```suggestion
            tag={{
              ...tag,
              title: `${tag.title} (${peoplePerTag[tag.id]})`
            }}
```

This of course doesn't _change_ the tag title, just the way it's rendered, without having to change the API of `TagChip`.